service/worker/scanner/workflow.go

```diff
@@ -54,10 +55,16 @@ const (
 	historyScannerWFTypeName     = "cadence-sys-history-scanner-workflow"
 	historyScannerTaskListName   = "cadence-sys-history-scanner-tasklist-0"
 	historyScavengerActivityName = "cadence-sys-history-scanner-scvg-activity"
+
+	executionsScannerWFID           = "cadence-sys-executions-scanner"
```

Comment:
These need to be renamed. @samarabbas, why did you leave lines 55-57?

Reply:
Will fix all of these in separate PR